tests/Tests/200__monitor_and_manage/220__grafana/220__test_verify_port_forwarding_grafana.robot

```diff
@@ -0,0 +1,48 @@
+*** Settings ***
+Documentation   Verify that MT-SRE can connect to Grafana using port-forwarding
+...
+...             = Variables =
+...             | PROC             | Required |        Is used  to get the process id|
+
+Library        OperatingSystem
+Library        Process
+Library        SeleniumLibrary
+
+
+*** Variables ***
+${PROC}     none
+
+
+*** Test Cases ***
+Verify that MT-SRE can connect to Grafana using port-forwarding
+    [Documentation]  Verify Grafana webiste is using port-forwarding
+    [Tags]   Tier2
+     ...     ODS-754
+
+    # Port Forwarding
+    ${PROC}    Start Process    oc -n redhat-ods-monitoring port-forward $(oc get pods -n redhat-ods-monitoring | grep grafana | awk '{print $1}' | head -n 1) 3001  shell=True  # robocop: disable
+    # It takes aroung 7 sec for the website to come up
+    Sleep  7s
+    ${response} =   Run   curl -I "http://localhost:3001"   # robocop: disable
```

Comment:
hm a question. If grafana needs to be accessed by outside, should the test use the grafana route instead of "localhost"?

Reply:
This is a special use case. I think MT-SRE is using a special software that requires this enabled